Updated for AFS Public Preview

tjvishnu opened the pull request:
Added more documentation on additional questions from customer interactions and CSS queries. Also updated to reflect public preview availability of Backup for Azure File Sync

azurecla commented:
Hi __@victorar__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <p>
        It looks like you're working at Microsoft (victorar). If you're full-time, we DON'T require a contribution license agreement.
    </p>
    <p>
        If you are a vendor, DO please sign the electronic contribution license agreement. It will take 2 minutes and there's no faxing! https://cla.azure.com.
    </p>

TTYL, AZPRBOT;

tysonn commented:
@cherylmc - Could you review this public PR from @victorar?

tysonn commented:
@cherylmc - Second notice, please review

cherylmc commented:
Thanks @victorar. Good catch! @Tyson, can you please merge this in?

tjvishnu committed:
Updated for AFS Public Preview

Added more documentation on additional questions from customer interactions and CSS queries. Also updated to reflect public preview availability of Backup for Azure File Sync
